packages/react-percy-config/src/defaults.js

```diff
@@ -1,4 +1,5 @@
 export default {
+  renderer: '@percy-io/percy-snapshot-render-react',
```

Comment:
The renderer to use is now an option on the Percy config. As of right now we still only support React, so this defaults to the React renderer. But this gives us flexibility in the future to support other frameworks like Vue by simply having users swap out this config option.